test/legacy/roles/scaleway_s3/tasks/aws_s3.yml

```
@@ -0,0 +1,19 @@
+- name: Create a local file
+  tempfile:
+    state: file
+    suffix: temp
+  delegate_to: localhost
+  register: tempfile
+
+- include: aws_s3_create.yml
```

include will be deprecated in the near future: use `import_tasks` instead (static import):
```suggestion
- import_tasks: aws_s3_create.yml
```